### PR TITLE
Upgrade libheif in Windows build

### PR DESCRIPTION
**Pull request by novomesk**

### CODE REVIEW

File: Telegram/build/prepare/prepare.py [hunk 1]
```diff
@@ -749,7 +749,9 @@ def runStages():
         -DENABLE_PLUGIN_LOADING=OFF ^
         -DWITH_LIBDE265=ON ^
         -DWITH_SvtEnc=OFF ^
+        -DWITH_SvtEnc_PLUGIN=OFF ^
```
[the remark this answers is not in the document]
Comment (novomesk) [reply Telegram/build/prepare/prepare.py:752]: They are created via plugin_option macro.

[the remark this answers is not in the document]
Comment (ilya-fedin) [reply Telegram/build/prepare/prepare.py:752]: hmm, it seemed to me that only the _PLUGIN variable is read through that macro :thinking:

[the remark this answers is not in the document]
Comment (novomesk) [reply Telegram/build/prepare/prepare.py:752]: The macro creates two variables.

option(WITH_${variableName}

And the second one:

option(WITH_${variableName}_PLUGIN

[the remark this answers is not in the document]
Comment (ilya-fedin) [reply Telegram/build/prepare/prepare.py:752]: Well, I trust you, apparently I've read it wrong

**Pull request merged**